src/backend/opencl/magma/getrf.cpp

```diff
@@ -219,23 +219,22 @@ magma_int_t magma_getrf_gpu(
             magma_getmatrix<Ty>(m-j*nb, nb, dAP(0,0), maxm, work(0), ldwork, queue);
 
             if (j > 0 && n > (j + 1) * nb) {
-                CLBLAS_CHECK(gpu_blas_trsm(
-                                 clblasRight, clblasUpper, clblasNoTrans, clblasUnit,
-                                 n - (j+1)*nb, nb,
-                                 c_one,
-                                 dAT(j-1,j-1), lddat,
-                                 dAT(j-1,j+1), lddat,
-                                 1, &queue, 0, nullptr, &event));
-
-                if (m > j * nb)  {
-                    CLBLAS_CHECK(gpu_blas_gemm( clblasNoTrans, clblasNoTrans,
-                                                n-(j+1)*nb, m-j*nb, nb,
-                                                c_neg_one,
-                                                dAT(j-1,j+1), lddat,
-                                                dAT(j,  j-1), lddat,
+                OPENCL_BLAS_CHECK(gpu_blas_trsm(OPENCL_BLAS_SIDE_RIGHT, OPENCL_BLAS_TRIANGLE_UPPER, OPENCL_BLAS_NO_TRANS, OPENCL_BLAS_UNIT_DIAGONAL,
```

Comment:
I believe they are aligned, but your IDE (or GitHub) is showing it differently because the lines are now quite long. I've split it into smaller lines to make it look better (also for the cases below).